spec/chartmogul/data_source_spec.rb

```diff
@@ -59,18 +59,26 @@
 
       data_sources[0].destroy!
 
-      new_data_sources = ChartMogul::Import::DataSource.all
+      new_data_sources = ChartMogul::DataSource.all
       expect(new_data_sources).to be_empty
     end
 
     it 'correctly raises errors on 422', uses_api: true do
-      expect { ChartMogul::Import::DataSource.create! }.to raise_error(ChartMogul::ResourceInvalidError)
+      expect { ChartMogul::DataSource.create! }.to raise_error(ChartMogul::ResourceInvalidError)
     end
 
     it 'correctly raises errors on 404', uses_api: true do
-      ds = ChartMogul::Import::DataSource.new
+      ds = ChartMogul::DataSource.new
       ds.send(:set_uuid, '1234-a-uuid-that-doesnt-exist')
       expect { ds.destroy! }.to raise_error(ChartMogul::NotFoundError)
     end
+
+    it 'retrieves existing data source matching uuid', uses_api: true do
+      ds = ChartMogul::DataSource.create!(name: 'TestDS')
+      ds.send(:set_uuid, 'ds_5ee8bf93-b0b4-4722-8a17-6b624a3af072')
```

Comment:
so the UUID is not randomly generated and I VCR won't know how to handle the request